source/guides/administrator.rst

```diff
@@ -115,6 +115,10 @@ Learn how to configure and administer Mattermost apps.
    :glob:
 
    /mobile/mobile-overview.rst
+   /mobile/use-prebuilt-mobile-apps.rst
+   /mobile/build-custom-mobile-apps.rst
+   /mobile/deploy-mobile-apps-using-emm-provider.rst
+   /mobile/consider-mobile-vpn-options.rst
    /mobile/mobile-appconfig.rst
    /mobile/mobile-hpns.rst
    /mobile/mobile-faq.rst
```

Comment:
Can we a TOC to the top of this FAQ doc to make it easier to scan questions and jump to answers?

Reply:
Absolutely!